forest/intake_loader.py

```diff
@@ -0,0 +1,395 @@
+'''
+Module for loading data from the Pangeo CMIP6n intake catalogue (usng the intake-esm
+loader) created by NCAR.
+'''
+
+from datetime import datetime
+from collections import namedtuple
+import functools
+
+import bokeh
+import intake
+import iris
+import numpy
+
+from forest import geo, gridded_forecast
+
+# Location of the Pangeo-CMIP6 intake catalogue file.
+URL = 'https://raw.githubusercontent.com/NCAR/intake-esm-datastore/master/catalogs/pangeo-cmip6.json'
+
+@functools.lru_cache(maxsize=64)
+def _get_intake_vars(
+        experiment_id,
+        table_id,
+        grid_label,
+        institution_id,
+        member_id):
+    """
+    Helper function to get a list of variables for this particular combination
+    of parameters. Function is cahced to reduce remote queries.
+    """
+    collection = intake.open_esm_datastore(URL)
+    cat = collection.search(
+        experiment_id=experiment_id,
+        table_id=table_id,
+        grid_label=grid_label,
+        institution_id=institution_id,
+        member_id=member_id,
+    )
+    var_list = cat.unique('variable_id')['variable_id']['values']
+    return var_list
+
+
+@functools.lru_cache(maxsize=16)
+def _load_from_intake(
+        experiment_id,
+        table_id,
+        grid_label,
+        variable_id,
+        institution_id,
+        activity_id,
+        member_id):
+    """
+    Load data from the pangeo CMIP6 intake catalogue.The arguments relate to
+    the CMIP6 parameters of a dataset. The CMIP6 reference is the ESGF servers
+    which can be accessed here:
+    https://esgf-index1.ceda.ac.uk/search/cmip6-ceda/
+    Function is cahced to reduce remote queries.
+    """
+    collection = intake.open_esm_datastore(URL)
+    print('opening catalogue')
```

Comment:
There are three of these `print` calls in this function that need to be removed.

Reply:
Removed at 565af5de1b5b3ab3fdd123ee00ae1f97ebfc01f0